firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-# Unreleased (21.3.1)
+# Unreleased (21.3.2)
+- [changed] Transactions previously required that every document read in a
+  transaction must also be written to. This requirement has been removed, such
+  that you can now read a document in transaction without writing to it.
+- [changed] Firestore now recovers connectivity more quickly in situations 
```

Comment:
Other entries in this changelog use "Firestore now recovers more quickly" (i.e. without "connectivity"). 

You might also consider rephrasing to avoid "Firestore now recovers ... where Firestore". Something like:

> Firestore now recovers more quickly once connections suffering packet loss return to normal.

... or something like that.